layer1/chain/src/chain_sc.mligo

```diff
@@ -0,0 +1,26 @@
+(*
+    This smart contract is used only as a placeholder to try the /chain/ library. 
+*)
+
+#include "chain.mligo"
+type chain_storage = chain
+type chain_parameter = 
+    | Receive of batch
+    | Remove  of index
+
+(* ENDPOINTS *)
+let main (action, store : chain_parameter * chain_storage) : operation list * chain_storage = 
+    match action with
+    | Receive b -> 
+        begin 
+        match store_batch (b, store) with
```

Comment:
They are interchangeable only because for a simple example smart contract using the type `chain`, there is nothing else to put in the storage, so the type `chain_storage` is an alias. When the lib is used in the real smart contract, the storage will probably contain additionnal info.
I'd rather keep the notion of `store` sepârate, to maintain the difference between the type `chain`defined in the lib and the `storage` defined for the smart contract.